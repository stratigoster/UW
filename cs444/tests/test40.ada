PACKAGE foo IS
  procedure plus (a,b,c : IN integer; d,e,f : OUT float; g,h,i : IN OUT boolean);
  procedure plus (a,b,c : OUT integer; d,e,f : IN float; g,h,i : IN OUT boolean);
-- this is not proper overloading!
-- we do not overload by IN/OUT modes
END foo;
