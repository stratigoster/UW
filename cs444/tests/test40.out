***ERROR***:3:82:	Conflicting declaration `plus:function (integer,integer,integer,float,float,float,boolean,boolean,boolean) return void'

------
Compilation error
------
